### ISSUE TITLE
[Accessibility][Headings and Labels - User Input]: No Heading level is defined on page.

### ISSUE THREAD

**Issue by ShellySoni** [microsoft/BotFramework-Composer#2094]
**User Experience:**
If headings are not implemented on the page then it would be very difficult for the user to navigate the main content of the page as headings and labels are used to describe sections of content and controls respectively.

**Test Environment:**
Environment: Chromium Edge (Anaheim) + Narrator
Build Version: 2004 (OS Build 19569.1000)​​
Microsoft Edge Version: 82.0.436.0 (Official build) canary (64-bit)​
Test URL: http://localhost:3000/home

**Repro Steps:**
1. Open URL: http://localhost:3000/home
2. "Bot Framework Composer" homepage will gets opened.
3. Select any of the previously created bot from "Recent Bots" or create one from the available Bots under 'Examples' and activate it.
4. Activate 'Headings' using accessibility insights for web or activate scan mode and then press 'H' key to look for headings defined on page.
5. Observe the issue.

**Actual:**
No Heading is defined on page.

**Note:** 
Same issue is repro within :Bot responses, Notifications, and About page.

**Expected:**
Heading should be defined on page.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0cf80872-800d-4643-90cf-0c1c3d3e6260}

**Attachment:**
~[MAS2.4.6_Bot Responses_User input_Notifications_About.zip](https://github.com/microsoft/BotFramework-Composer/files/4253826/MAS2.4.6_Bot.Responses_User.input_Notifications_About.zip)~

[edited by @corinagum]
![MAS2 4 6_Bot Responses_User input_Notifications_About](https://user-images.githubusercontent.com/14900841/75380629-3a008600-588c-11ea-84bf-3d0fa3c905ac.png)

Attaching image so devs don't need to download the .zip file.

**Comment by corinagum** [comment]
![MAS2 4 6_Bot Responses_User input_Notifications_About](https://user-images.githubusercontent.com/14900841/75380629-3a008600-588c-11ea-84bf-3d0fa3c905ac.png)

Attaching image so devs don't need to download the .zip file.

**Comment by corinagum** [comment]
Can we also get a11y tags added to this issue? Including severity :)

**Comment by ashish315** [comment]
As checked and verified on below environment, the issue is fixed now. So, we are closing the issue.
**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)
